department-of-veterans-affairs/va.gov-cms
# VACMS-000: Disable layout builder in favor of fields | Remove Layout Builder

## Description
Lisa Trombley reported event listing pages showing broken block messages below introduction content. This was caused by layout builder being enabled on said content type. This pr undoes this unintentional config change.

Also removing Layout Builder and dependencies (per discussion with @oksana-c )

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/109198520-50c20d00-776c-11eb-8b59-1cc347163aed.png)


## QA steps
Go to `altoona-health-care/events` and visually verify that broken block messages do not appear

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
